3-probability/02-cond-prob-indep/ps.qmd

```diff
@@ -31,7 +31,7 @@ format: stat20handout-pdf
 
 #### Question 3 Bella and her toys
 
-My dog Bella has three toys that she loves: A stuffed and squeaky octopus, an orange ball, and a thick rope. One day we hid the octopus (too squeaky!) so she had to pick the ball or the rope. Each time she picks a toy, she chooses it independently of all the other times (like a coin toss). That day, she was busy, so went to her toys three times. 
+My dog Bella has three toys that she loves: A stuffed and squeaky octopus, an orange ball, and a thick rope. One day we hid the octopus (too squeaky!) so she had to pick the ball or the rope and she was equally likely to choose each of them. Each time she picks a toy, she chooses it independently of all the other times (like a coin toss). That day, she was busy, so went to her toys three times. 
```

Comment:
@shomurali Isn't this needed to determine independence? That's to say, are there certain probabilities of these outcomes that would induce some dependence?

Reply:
yes, equally likely is needed for independence. Did I delete it? oops.

Reply:
I mean, it is a coin toss, so not sure we need it to be equally likely, but I meant it to be equally likely, so they could use the \Omega they had written out many times by now (one hopes)